src/platform-implementation-js/views/route-view/route-view.ts

```diff
@@ -22,9 +22,9 @@ class RouteView extends EventEmitter {
   }
 
   /**
-   * @returns a string of the ID of the RouteView. This is the same routeID that you give Router.goto() or Router.createLink(). This will be a value from NativeRouteIDs.
+   * @returns a string or string[] of the ID of the RouteView. This is the same routeID that you give Router.goto() or Router.createLink(). This will be a value from NativeRouteIDs.
    */
-  getRouteID(): string {
+  getRouteID(): string | string[] {
```

Comment:
This change would be an API breaking change (applications can no longer pass the return value of this to Router.goto() or Router.createLink()) but also it's unnecessary: a RouteView represents the URL the user is currently at and they will only be at one route at a time. The changes in this file and route-view-driver.ts should be discarded.